components/server/src/migration/user-to-team-migration-service.spec.db.ts

```diff
@@ -1,20 +1,28 @@
 /**
- * Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
  * Licensed under the GNU Affero General Public License (AGPL).
  * See License.AGPL.txt in the project root for license information.
  */
 
 import * as chai from "chai";
-import { testContainer } from "./test-container";
-import { TypeORM } from "./typeorm/typeorm";
 import { UserToTeamMigrationService } from "./user-to-team-migration-service";
-import { UserDB } from "./user-db";
-import { TeamDB } from "./team-db";
-import { ProjectDB } from "./project-db";
 import { v4 as uuidv4 } from "uuid";
-import { WorkspaceDB } from "./workspace-db";
+import { ProjectDB, TeamDB, TypeORM, UserDB, WorkspaceDB, testContainer } from "@gitpod/gitpod-db/lib";
+import { ContainerModule } from "inversify";
+import { StripeService } from "../user/stripe-service";
 const expect = chai.expect;
 
+const mockedStripe = new StripeService();
+mockedStripe.updateAttributionId = async (stripeCustomerId: string, attributionId: string) => {
+    return;
+};
```

Comment:
In other places we grab the service from the test container, and then override rather than constructing differently. Is that not possible here?

Reply:
Yes, it is not possible because it would pull in transitive dependencies which we don't have. This makes sure we stop dependency resolution.